config/webpack.config.babel.js

```diff
@@ -150,6 +150,7 @@ export default {
     path: path.resolve(context, PUBLIC_DIR),
     crossOriginLoading: 'anonymous',
     publicPath: ASSETS_ROOT,
+    sourceMapFilename: production ? '[name].[chunkhash].js.map' : '[name].js.map',
```

Comment:
So what are we doing now with the source map? It definitely shouldn't lay open for everyone in the public folder.